src/api/emails/submission-deadline/index.test.ts

```diff
@@ -0,0 +1,48 @@
+import { submissionDeadlineEmail } from '.';
+import notify from '../../integrations/notify';
+import { EMAIL_TEMPLATE_IDS } from '../../constants';
+import { mockApplication, mockSendEmailResponse } from '../../test-mocks';
+import { SubmissionDeadlineEmailVariables } from '../../types';
+
+describe('emails/submission-deadline', () => {
+  const sendEmailSpy = jest.fn(() => Promise.resolve(mockSendEmailResponse));
+
+  const templateId = EMAIL_TEMPLATE_IDS.APPLICATION.SUBMISSION.DEADLINE;
+
+  const variables = {
+    email: mockApplication.owner.email,
+    name: `${mockApplication.owner.firstName} ${mockApplication.owner.lastName}`,
+    referenceNumber: String(mockApplication.referenceNumber),
+    applicationUrl: '',
+    buyer: String(mockApplication.buyer.companyOrOrganisationName),
```

Comment:
`buyerName`?

Do we need `applicationUrl` and if so should it be an empty string?

maybe could destructure `mockApplication` here as well for `owner, `buyer` etc